<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>250</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="companyTextBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>155</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="surnameTextBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>195</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="nameTextBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>15</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>195</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="contactLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="IDTextBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>85</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="companyLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>155</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company:</string>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>50</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Surname:</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>85</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID Number:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="contactTextBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameTextBox</tabstop>
  <tabstop>surnameTextBox</tabstop>
  <tabstop>IDTextBox</tabstop>
  <tabstop>contactTextBox</tabstop>
  <tabstop>companyTextBox</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
